src/py/aspen/database/models/sample.py

```diff
@@ -123,6 +123,11 @@ class Sample(idbase, DictMixin):
             }
         },
     )
+    region = Column(
+        String,
```

Comment:
would it make more sense to make this an enum field instead?

Reply:
I looked at the region data for all the gisaid data and it appears very constrained to the continents.  I think it's pretty safe to make it an enum.